benchmark/performance_utils.py

```diff
@@ -52,8 +52,7 @@ def SkipVersion(module_name, skip_pattern):
         raise ValueError("Cannot parse version number from skip_pattern.")
 
     try:
-        module = importlib.import_module(module_name)
-        version = module.__version__
+        version = importlib.metadata.version(module_name)
```

Comment:
Actually, a package's __version__ is arbitrarily defined. It may not agree with the version number of the package.